client/src/types/Trip.ts

```diff
@@ -4,6 +4,11 @@ export type Trip = {
   country: string;
   startDate: Date;
   endDate: Date;
-  stops: string[];
   details: GeoJSON.FeatureCollection;
 };
+
+export type Stop = {
+  type: string;
+  geometry: { type: string; coordinates: number[] };
+  properties: { name: string; label: string };
+};
```

Comment:
this will help me solve my problem ;-)

Reply:
Don't think this is the final version, but it's a step in the direction